src/commands/goal/set.ts

```diff
@@ -43,6 +43,7 @@ export const GoalSetCommand: SubCommand = {
     ]
   }),
   execute: async (interaction: CommandInteraction, guildConfig: GuildConfig) => {
+    await UserConfig.findOrCreate(interaction.user.id)
```

Comment:
Do we want to make sure we have a `UserConfig` created for every command? that seems to me like it's be the safer way to go about it?

In that case I'd suggest putting this in [`events/interactions.ts`](https://github.com/aigeroni/Winnie_Bot/blob/main-2.0/src/events/interaction.ts) like we do `GuildConfig`

Reply:
That might be the better idea.  Let me move it.